src/components/reps/value-renderer.jsx

```diff
@@ -122,39 +111,34 @@ const complexHandlers = [
   // matrix must come before array!
   matrixHandler,
   arrayHandler,
-  dateHandler,
-  functionHandler,
   errorHandler,
-  domElementHandler,
   promiseHandler,
   defaultHandler,
 ].map(h => wrapHandler(h))
 
-let handlers = simpleTypeHandlers.concat(complexHandlers)
-
+let handlers = simpleHandlers.concat(complexHandlers)
 
 const userHandlers = []
 
 export function addOutputHandler(handler) {
   // insert new handlers *after* the scalar handlers
-  userHandlers.unshift(wrapHandler(handler))
-  handlers = simpleTypeHandlers.concat(userHandlers, complexHandlers)
+  userHandlers.unshift(handler)
```

Comment:
why'd you remove the wrapHandler here? don't we especially need to wrap user-added handlers in try/catch?

Reply:
Sorry -- this was just left in after experimenting with something...